voucher/urls.py

```diff
@@ -3,8 +3,11 @@
 
 # SharedAPIRootRouter is automatically imported in global urls config
 router = SharedAPIRootRouter()
-router.register(r'voucher/wallets', WalletViewSet, base_name='voucher_wallets')
+router.register(r'voucher/wallets/voucher', VoucherWalletViewSet, base_name='voucher_wallets')
+router.register(r'voucher/wallets/coffee', CoffeeWalletViewSet, base_name='coffee_wallets')
+router.register(r'voucher/registerlogs/voucher', WorkLogViewSet)
+router.register(r'voucher/registerlogs/coffee', RegisterLogViewSet)
+router.register(r'voucher/uselogs/voucher', VoucherUseLogViewSet)
+router.register(r'voucher/uselogs/coffee', CoffeeUseLogViewSet)
```

Comment:
Støtter i utgangspunktet det @matsasc sier. Men her burde vi sikkert rydde opp litt uansett mens vi har muligheten. Dog synes jeg at `"action"/model` er gal fremgangsmåte, og at det heller burde være "modellen" man opererer over som går først.

Jo mer jeg tenker over det lurer jeg på om man kanskje burde laget vouchers/coffe mer generisk, i stedet for å splitte det i to modeller. Hadde det vært mulig? Typisk at en wallet får en type.